erigon-lib/common/dbg/dbg_ctx.go

```diff
@@ -0,0 +1,23 @@
+package dbg
+
+import (
+	"context"
+)
+
+type debugContextKey struct{}
+
+// Enabling detailed debugging logs for given context
+func ToContext(ctx context.Context, v bool) context.Context {
```

Comment:
```suggestion
func ContextWithDebug(ctx context.Context, v bool) context.Context {
```

Reply:
then in code it will look like `dbg.ContextWithDebug` - 2 times "debug"